.pipelines/containerSourceData/base/Dockerfile-Base-Template

```diff
@@ -29,4 +29,8 @@ COPY $EULA .
 COPY --from=builder /staging/etc/ /etc/
 COPY --from=builder --chown=${USER_UID}:${USER_GID} /staging/home/ /home/
 
+FROM scratch
```

Comment:
Wow .. I wonder if this saves us a bunch of space!

Reply:
Sadly, no. We are just making our core images have a single layer. The additional layers previously were **not** contributing to the size so much that we would see a noticeable difference with this change.